src/Enzo/enzo_constants.hpp

```diff
@@ -45,6 +45,8 @@ namespace enzo_constants {
   const double clight = 29979245800.0;
 
   // Gravitational constant in CGS
+  // Note: this value is ovewritten by Method:gravity:grav_constant in non-cosmological
+  // simulations.
```

Comment:
```suggestion
  // Note: in non-cosmological simulations, the Gravity solver uses the value specified by 
  // Method:gravity:grav_constant
```
Minor suggestion (to avoid the word `overwrite`), feel free to ignore this.